src/inquirer/render/console/_checkbox.py

```diff
@@ -65,14 +65,16 @@ def get_options(self):
             yield choice, selector + " " + symbol, color
 
     def process_input(self, pressed):
+        # Remove after merge #83
```

Comment:
You are referencing a #83 here probably from other project since it is completely unrelated to what we have here. For referencing other projects you should put the whole link (https://github.com/magmax/python-readchar/pull/83) to avoid confusion of developers reading this code.